install_config/install/deploy_router.adoc

```diff
@@ -498,6 +561,74 @@ $ modshard 3 \
     ROUTE_LABELS='sla=low'
 ----
 
+[[using-namespace-router-shards]]
+=== Using Namespace Router Shards
+
+Each
+link:../../architecture/core_concepts/projects_and_users.html#projects[project has its own namespace].
+The routes for a project can be handled by a selected router by using
+`NAMESPACE_LABELS`.  The router is given a selector for a `NAMESPACE_LABELS`
+label and the project that wants to use the router applies the `NAMESPACE_LABELS`
+label to its namespace.
```

Comment:
Sorry for the delay - got sidetracked and then it was pain doing the markup in here.
So I just created a gist for the content.
Please see:  https://gist.github.com/ramr/586a16ede5403357a4709fd01927614a

Let me know if that clarifies things or then we can probably have a call since that might make it clearer.